sample/src/seeder.ts

```diff
@@ -1,4 +1,4 @@
-import { seeder } from 'nestjs-seeder';
+import { seeder } from '../../lib';
```

Comment:
This breaks the sample build. When we build the sample directory using 
`cd sample && npm run start:dev`
It should not reference the parent folder in any way, this will cause the typescript compiler to change the root directory from `sample` directory to `nestjs-seeder` directory. Besides, it's not a sample when it's used differently in the user project (The user project will not include `lib` directory).

Using `npm pack` or `npm link` for testing the library is more appropriate.

Note: No need to update the PR, I'll update it for you :)